_opensearch/query-dsl/text-analyzers.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title: Text analyzers
+parent: Query DSL
+nav_order: 41
+---
+
+
+# Text analyzers
+
+OpenSearch applies text analysis during indexing or searching for `text` fields. There is a standard  analyzer that OpenSearch uses by default for text analysis. To optimize unstructured text for search, you can convert it into structured text with our specialized text analyzers.
+## Specialized analyzers
+
+OpenSearch provides specialized analyzers to convert your structured text into the format that works best for your searches.
+
+OpenSearch supports the following specialized text analyzers:
+
+1. **Standard analyzer** – parses text strings into terms at word boundaries per the Unicode text segmentation algorithm. It also removes stop words and removes punctuation and lowercase terms.
```

Comment:
Standard analyzer does not remove stop words by default. It **can** remove stop words if you turn on an option on the analyzer. It removes most but not all punctuation. It also converts all characters to lower case.

Reply:
I think you meant "removes punctuation and lowercase**s** terms"

Reply:
thanks for clarifying info